docs/PLUGIN_DIAGNOSTICS.md

```diff
@@ -0,0 +1,114 @@
+# Plugin Diagnostics
+
+Plugin diagnostics provides a simple way to verify that a plugin is capable of running without requiring the plugin to be loaded and a task started.
+
+It is possible to run plugin diagnostics without any arguments, e.g.:
```

Comment:
Add here notice that if the plugin has some configuration needed, the defaults will be taken.